ui/src/app/kvm/components/KVMResourcesCard/_index.scss

```diff
@@ -5,10 +5,34 @@
   .kvm-machine-list-modal {
     max-width: none;
     min-width: 0;
-    width: 35rem;
+    width: calc(100vw - #{$sp-unit * 10});
```

Comment:
Why offset by 80?

Reply:
This was laziness hahaha. The offset is the sum of the gutters on the left and right on x-small screens + the inner horizontal padding of the card. That way the dropdown centers nicely in the card on small screens. I can update with vanilla variables